policy/modules/kernel/files.fc

```diff
@@ -297,6 +297,7 @@ ifndef(`distro_redhat',`
 /var/run/.*			gen_context(system_u:object_r:var_run_t,s0)
 /var/run/.*\.*pid		<<none>>
 /var/run/lock/.*		<<none>>
+/var/run/motd.d(/.*)?		gen_context(system_u:object_r:etc_t,s0)
```

Comment:
i'm wondering if the spacing should be changed to align `gen_content` with the `<<none>>` on the line above

Reply:
On GitHub on my end, the spacing of `gen_context` looks to be in line with the `<<none>>` on line 299. Maybe it using tabs is causing it to display differently?

Reply:
ahh yes. the dreaded varying display width for a tab :). I pulled down the patch and applied it locally and changed my tabwidth to 8 and it looks good.

ignore me @rfairley :smile:

Reply:
Thanks and no problem :) had that hit me on linux-pam as well which looked good only with a width of 8